sdk/python/kfp/dsl/_placeholders.py

```diff
@@ -0,0 +1,13 @@
+FLAGS = {
+			'EXECUTION_ID' : '{{workflow.uid}}-{{pod.name}}',
+			'POD_NAME': '{{pod.name}}',
+			'RUN_ID' : '{{workflow.uid}}',
+			'RUN_EXIT_STATUS' : '{{workflow.status}}',
+			'RUN_NAME' : '{{workflow.name}}',
+			'RUN_CREATION_TIMESTAMP' : '{{workflow.creationTimestamp}}',
+			'RUN_NAMESPACE' : '{{workflow.namespace}}',
+			'RUN_DURATION' : '{{workflow.duration}}'
+		}
+
+def get_run_information_placeholder(flag_name):
```

Comment:
Do we really need this function? We can just expose the placeholders as constants. (Like `EXECUTION_ID_PLACEHOLDER` in `__init__.py`)

Reply:
I think the function works better as an abstraction. It is not guaranteed that we will always be able to access the underlying orchestrator's data as simple templated variables, it might be that we have to call several low-level functions, or even api calls to extract the information for instance. I don't think that would work for some variables like `RUN_DURATION` or `RUN_EXIT_STATUS ` which cannot be constants.

Reply:
The function call can be confusing since users might think that they actually "call several low-level functions, api calls to extract the information". Instead, the placeholders are put in at compile-time and final resolving only happens at runtime.

I think that `RUN_EXIT_STATUS_PLACEHOLDER` can be a constant *placeholder* object, which will be replaced with the real value at runtime.

Reply:
So you are saying that we can get away with the placeholder objects since they could end up being complex objects that contain API calls on their `__str__` function, right?